excellent/tests.go

```diff
@@ -183,22 +183,18 @@ func HasRunStatus(env utils.Environment, args ...interface{}) interface{} {
 		return fmt.Errorf("HAS_RUN_STATUS takes exactly two arguments, got %d", len(args))
```

Comment:
I've left the test names as they are for now instead of merging into a single `has_status`. Seems like there is some utility in knowing what is being tested, and potentially validating the different statuses that are applicable to runs vs webhooks